ext_emconf.php

```diff
@@ -11,14 +11,14 @@
     'title' => 't3oodle',
     'description' => 'Simple poll extension for TYPO3 CMS. t3oodle allows your frontend users to create new polls and vote for existing ones.',
     'category' => 'plugin',
-    'version' => '0.9.2',
+    'version' => '0.10.0',
     'state' => 'beta',
     'author' => 'Armin Vieweg',
     'author_email' => 'info@v.ieweg.de',
     'author_company' => 'FGTCLB',
     'constraints' => [
         'depends' => [
-            'typo3' => '9.5.0-10.4.99',
+            'typo3' => '10.4.0-11.5.99',
```

Comment:
composer.json has requirement only for TYPO3 v11, I guess, we should do the same here.

Additionally, dependencies to other extensions are missing, (fluid_styled_content, numbered_pagination). For legacy installations respecting the dependencies, these should be added here, too.